inc/boost/functional.hpp

```diff
@@ -21,7 +21,6 @@ namespace boost
     namespace functional
     {
         namespace detail {
-#if defined(_HAS_AUTO_PTR_ETC) && !_HAS_AUTO_PTR_ETC
```

Comment:
The commenting out of this line and of the _else_ code below are the only significant changes to this file.  The other 28 changes are simply whitespace changes.

I don't know if you are adopting the philosophy "as long as I'm changing this file, I may as well eliminate extraneous whitespace", or if your editor simply does this automatically.  Mainly, I just want to make sure you are aware of this.

Fortunately, both my editor's ediff utility and GitHub's "Diff view" settings contain options to hide whitespace changes, so this isn't too annoying.

Reply:
My editor does this automatically, to enforce BioCro style guidelines about trailing whitespace.